src/scripts/fixtures_script.sql

```diff
@@ -0,0 +1,61 @@
+-- FUNCTION : product_fixture
+
+CREATE PROCEDURE build_product(
```

Comment:
Could you rename it to something more meaningful like `insert_product_with_default` perhaps?

Reply:
done